db/seeds.rb

```diff
@@ -5,3 +5,36 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+require 'faker'
+
+puts "will create trainer..."
+trainer = User.create!({
+  email: "trainer@mewbnb.com",
+  password: "password123"
+})
+
+puts "will create gym_leader..."
+gym_leader = User.create!({
+  email: "gymleader@mewbnb.com",
+  password: "password123"
+})
+
+
+10.times do
+  puts "will create pokemon..."
+  pokemon = Pokemon.create!({
+    name: Faker::Games::Pokemon.name,
+    rate: rand(50..500),
+    description: Faker::Games::Pokemon.move,
+    location: Faker::Games::Pokemon.location,
+    user: gym_leader
+  })
+  puts "will create booking..."
+  Booking.create!({
+    user: trainer,
+    pokemon: pokemon,
+    status: ["pending", "confirmed", "rejected"].sample,
```

Comment:
I think that this is the status of the booking, not the request. 
So it should probably be "available", "request" and "booked".
What do you think?